pkgs/applications/graphics/kcc/default.nix

```diff
@@ -1,39 +1,48 @@
-{ lib
-, mkDerivationWith
-, python3Packages
-, fetchPypi
-, p7zip
-, archiveSupport ? true
-}:
-
-mkDerivationWith python3Packages.buildPythonApplication rec {
+{ stdenv, lib, qt6, python3Packages, fetchFromGitHub, p7zip
+, archiveSupport ? true }:
+python3Packages.buildPythonApplication rec {
   pname = "kcc";
-  version = "5.5.1";
+  version = "6.2.2";
 
-  src = fetchPypi {
-    inherit version;
-    pname = "KindleComicConverter";
-    sha256 = "5dbee5dc5ee06a07316ae5ebaf21ffa1970094dbae5985ad735e2807ef112644";
+  src = fetchFromGitHub {
+    owner = "ciromattia";
+    repo = "kcc";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-GX0WPu1EZx/8H1ajivxgnym4jECmwPBJU25vzcfOQCw=";
   };
 
-  propagatedBuildInputs = with python3Packages ; [
+  nativeBuildInputs = [ qt6.wrapQtAppsHook ];
```

Comment:
I think you can update the call to `libsForQt5.callPackage` to just be `callPAckage` for `kcc` in `all-packages.nix` now that you've added this.

You could just drop that line in `all-packages.nix` and move the package to `pkgs/by-name/kc/kcc/package.nix`, which is something that should probably be done eventually anyways.